### PR TITLE
🚚 #384: move /public folder to root

### PR DESCRIPTION
**Pull request by samrichca**
### Consent-UI
- Move /public folder from /src to root folder to fit NextJS spec
- Update image imports
  - Image URLs are no longer hashed
  - Remove placeholder=blur on the homepage background, it's a small image & placeholder wasn't used correctly.
- Add placeholder PDFs for consent wizard step 1 so we can make sure PDFs can be accessed by their URL
- Didn't update i18n routing middleware because the `assets` folder is already excluded.

## PR Readiness Checklist

- [x] "Expected Outcome(s)" in ticket have been met
- [x] Ticket number included in PR title
- [x] Connected ticket to PR
- [x] Labels added to PR for service name (`consent-api`, `data-mapper`, etc...), type (`chore`, `documentation`, etc...), status (`draft`, `on-hold`, etc...) **if applicable**
- [x] Manual testing completed
- [x] Builds locally without errors or warnings
- [x] Tests are updated (if required) and passing
- [x] PR feedback has been addressed **if applicable**
- [x] New environment variables added to `.env.schema` files, `README.md`
- [x] Added copyrights to new files

### CODE REVIEW

File: apps/consent-ui/src/components/views/Home/index.tsx [hunk 1]
```diff
@@ -32,7 +31,7 @@ const HomeComponent = async ({ currentLang }: { currentLang: ValidLanguage }) =>
 	return (
 		<div className={styles.heroContainer}>
 			<div className={styles.backgroundImg}>
-				<Image src={BackgroundImage} alt="" priority placeholder="blur" />
+				<Image alt="" fill priority sizes="100vw" src="/assets/images/landing-page.jpg" />
```
Comment (samrichca) [inline apps/consent-ui/src/components/views/Home/index.tsx:34]: `placeholder="blur"` wasn't used properly and this image is only 50kb

File: apps/consent-ui/src/components/common/Header/index.tsx [hunk 2]
```diff
@@ -18,36 +18,28 @@
...
 const Header = async ({ currentLang }: { currentLang: ValidLanguage }) => {
 	const translate = getTranslation(currentLang);
-	const icon = icons[currentLang || defaultLanguage];
 	return (
 		<HeaderWrapper currentLang={currentLang}>
 			<div>
 				<Link href={`/${currentLang}`}>
 					<Image
-						src={icon}
+						src="/assets/images/ohcrn_large.svg"
```
Comment (samrichca) [inline apps/consent-ui/src/components/common/Header/index.tsx:37]: for translation we could do ```src={`/assets/images/ohcrn_large_${currentLang}.svg`}```

I don't think we need the object on line 32

File: apps/consent-ui/src/components/layouts/SideImageLayout/index.tsx [hunk 3]
```diff
@@ -91,7 +97,13 @@ const SideImageLayout = ({
 									<span>{desktopNavAction.topText}</span>
 									<span className={styles.bottomText}>{desktopNavAction.bottomText}</span>
 								</div>
-								<Image src={ChevronSvg} alt="" className={styles.chevron} />
+								<Image
+									alt=""
```
Comment (anncatton) [inline apps/consent-ui/src/components/layouts/SideImageLayout/index.tsx:101]: Not in this pr, but we should set up a dictionary for alt text for the icons that require it, just so it's in place.

**Review by anncatton — COMMENTED** [review]
A couple comments - also, is there a way to test out the placeholder pdfs?

File: apps/consent-ui/src/components/common/Header/index.tsx (hunk 2 above)
Comment (samrichca) [reply apps/consent-ui/src/components/common/Header/index.tsx:37]: yup. renamed a random image to the FR filename

image

image

File: apps/consent-ui/src/components/views/Home/index.tsx (hunk 1 above)
Comment (samrichca) [reply apps/consent-ui/src/components/views/Home/index.tsx:34]: I think that was before I asked Patrick to resize the images, hopefully it'll be OK in a real env. if not we should add actual blurred placeholders

updated the CSS to get rid of the warning, tested up to 4000px screen size

**Comment by samrichca** [comment]
@anncatton to test the placeholder PDFs, on this branch, go to http://localhost:3000/assets/pdfs/PLACEHOLDER_study-consent_EN.pdf

File: apps/consent-ui/src/components/views/Dashboard/index.tsx [hunk 4]
```diff
@@ -39,7 +38,7 @@ const DashboardComponent = async ({ currentLang }: { currentLang: ValidLanguage
 			<DashboardNotification currentLang={currentLang} />
 			<Card dropShadow="sm" className={clsx(styles.card, styles[consentStatus])} layout="row">
 				<div className={styles['consents-img']}>
-					<Image src={ConsentsImage} alt="" />
+					<Image width={2883} height={1922} src="/assets/images/consents.jpeg" alt="" />
```
Comment (anncatton) [inline apps/consent-ui/src/components/views/Dashboard/index.tsx:41]: this image requires the `priority` prop

**Review by anncatton — COMMENTED** [review]
@samrichca i'm rethinking this approach. I see the need for moving the public folder for the pdfs, and I like the organization of the assets folders. But I feel like it's better to maintain the static import approach for the image files, the hash values are consistent between builds so caching should be fine. 

Those imports should still work with the public folder being outside `src`:

```
// Dashboard/index.ts

import consentsImg from 'src/../public/assets/images/consents.jpeg'
// OR
import consentsImg from '../../../../public/assets/images/consents.jpeg'
```

It should also let us take advantage of some of the image optimization, and remove the need for all the explicit dimensions props!

File: apps/consent-ui/src/components/views/Dashboard/index.tsx (hunk 4 above)
Comment (samrichca) [reply apps/consent-ui/src/components/views/Dashboard/index.tsx:41]: ok, I'll close this PR and just add the PDFs in the consent step 1 PR

Comment (anncatton) [reply apps/consent-ui/src/components/views/Dashboard/index.tsx:41]: i don't think that's necessary! - we'd still want the move for the public folder here. would just need to change the img imports back to the `import imageName from './imageSrc'`

File: apps/consent-ui/src/components/common/Footer/Left.tsx [hunk 5]
```diff
@@ -21,31 +21,38 @@ import Link from 'next/link';
...
 const Left = ({ translate }: { translate: TranslationFunction }) => {
 	return (
 		<div className={styles.left}>
 			<Link href="#" className={styles.icon}>
-				<Image src={OICRLogo} alt={translate('footer', 'oicrLogoAlt')} className={styles.oicr} />
+				<Image
+					alt={translate('footer', 'oicrLogoAlt')}
+					className={styles.oicr}
+					height={83}
+					src="/assets/images/oicr.svg"
```
Comment (xusandrew) [inline apps/consent-ui/src/components/common/Footer/Left.tsx:35]: random question, why were the imports for the svgs removed in favour of directly putting the relative paths in the src attribute? was it convention or personal preference?

Comment (samrichca) [reply apps/consent-ui/src/components/common/Footer/Left.tsx:35]: that's the convention when using the top level public folder

File: apps/consent-ui/src/components/views/Dashboard/index.tsx [hunk 6]
```diff
@@ -39,7 +39,7 @@ const DashboardComponent = async ({ currentLang }: { currentLang: ValidLanguage
 			<DashboardNotification currentLang={currentLang} />
 			<Card dropShadow="sm" className={clsx(styles.card, styles[consentStatus])} layout="row">
 				<div className={styles['consents-img']}>
-					<Image src={ConsentsImage} alt="" />
+					<Image src={ConsentsImage} alt="" priority />
```
Comment (samrichca) [inline apps/consent-ui/src/components/views/Dashboard/index.tsx:42]: I got a console warning about this image being above the fold and not having a `priority` flag

File: apps/consent-ui/src/components/views/Home/Home.module.scss [hunk 7]
```diff
@@ -4,6 +4,8 @@
 }
 
 .backgroundImg {
+	position: absolute;
```
Comment (samrichca) [inline apps/consent-ui/src/components/views/Home/Home.module.scss:7]: images with the `fill` property need parent element that isn't `position: static`

**Review by anncatton — APPROVED** [review]

**Pull request merged**